react-renderer/tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "outDir": "build/dist",
     "module": "esnext",
     "target": "es5",
-    "lib": ["es5", "es6", "es7", "dom"],
+      "lib": ["es5", "es6", "es7", "esnext", "dom"],
```

Comment:
FYI I added esnext here @wodeni @maxkrieger. Wanted to use `Object.fromEntries` in `EngineUtils`. Lmk if unnecessary

Reply:
Lgtm

Reply:
Not sure how the polyfill coverage is but shouldn't be a problem